documentation.html.haml

```diff
@@ -9,7 +9,18 @@ title: Get Started
     %p Get up and running with Infinispan quickly. Let us walk you through it!
 
 .row-fluid
-  .span5.quickstarts
+  .span5.tutorials
+    %h2 Tutorials
+    %table.table.table-bordered
+      %tbody
+      %tr
+        %td
+          %i.icon-certificate.pull-left
+          %b The Weather App
+          %br
+          %i a tutorial on how to embed Infinispan in your application
+        %td          
+          %a{:href=>"/tutorials/embedded/"} Tutorial
```

Comment:
It would be nice to have a link pointing to it from the "Getting Started Guide" as well (there are links from there to the User Guide and quickstarts but this is definitely a good starting point as well).